haas/api.py

```diff
@@ -510,6 +510,23 @@ def headnode_detach_network(headnode, hnic):
                             # Network Code #
                             ################
 
+@rest_call('GET', '/networks')
+def list_networks():
+    """List all networks.
+    Returns a JSON array of strings representing a list of projects.
+    Example:  {"channels": ["101", "107"], "name": ["net-05", "qnet-00"]}
+    """
```

Comment:
This docstring isn't right.  Currently, you return a JSON dictionary, indexed by the network names, whose values are dictionaries.  The example is also incorrect.